usr/share/rear/output/USB/Linux-i386/850_make_USB_bootable.sh

```diff
@@ -1,3 +1,8 @@
+# Only run for syslinux and extlinux
+if [ ! -z $USB_BOOTLOADER ] && [[ ! $USB_BOOTLOADER =~ syslinux|extlinux ]]; then
```

Comment:
I do not see where USB_BOOTLOADER is set to syslinux or extlinux.
Similar as https://github.com/rear/rear/pull/2655/files#r672040064
I think the test should be simplified to
```
# Skip if USB_BOOTLOADER is 'grub'
# see its counterpart in output/USB/Linux-i386/300_create_grub.sh
test "$USB_BOOTLOADER" = "grub" && return
```

Reply:
it does not need to be set since empty is previous behaviour but may be set by config